lib/home/profile/info/heart_center_info.dart

```diff
@@ -34,219 +30,169 @@ class _HeartCenterInfoState extends State<HeartCenterInfo> {
-                      validator: (value) {
-                        if (value == null || value.isEmpty) {
-                          return 'Please enter your email address';
-                        }
-                        if (!RegExp(r'^[^@]+@[^@]+\.[^@]+').hasMatch(value)) {
-                          return 'Please enter a valid email address';
-                        }
-                        return null;
-                      },
-                    ),
-                    const SizedBox(height: 8.0),
-                    TextFormField(
-                      controller: _messageController,
-                      decoration: const InputDecoration(labelText: 'Your Message'),
-                      maxLines: 4,
-                      validator: (value) {
-                        if (value == null || value.isEmpty) {
-                          return 'Please enter a message';
-                        }
-                        return null;
-                      },
-                    ),
-                    const SizedBox(height: 16.0),
-                    ElevatedButton(
-                      onPressed: () {
-                        if (_formKey.currentState!.validate()) {
-                          _submitForm(
-                            _emailController.text,
-                            _messageController.text,
-                          );
-                          ScaffoldMessenger.of(context).showSnackBar(
-                            const SnackBar(content: Text('Form submitted successfully!')),
-                          );
-                          _emailController.clear();
-                          _messageController.clear();
-                        }
-                      },
-                      child: const Text('Submit'),
-                    ),
-                  ],
+          SizedBox(height: 32),
+          ElevatedButton(
+            onPressed: HeartCenterInfo.donate,
+            child: Text('Donate'),
+          ),
+        ],
+      1 => [
+          const Text('Our Intention', style: StyleText(size: 24, weight: 650)),
+          const SizedBox(height: 16),
+          Text(aboutUs.replaceAll(' \n', ' '), textAlign: TextAlign.center),
+        ],
+      2 || _ => [
+          const Text(
+            'The Heart Center Inc',
+            style: StyleText(size: 24, weight: 650),
+          ),
+          const SizedBox(height: 16),
+          Form(
+            key: _formKey,
+            child: Column(
+              children: [
+                TextFormField(
+                  controller: _emailController,
+                  decoration: const InputDecoration(labelText: 'Your Email'),
+                  keyboardType: TextInputType.emailAddress,
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter your email address';
+                    }
+                    if (!EmailValidator.validate(value)) {
```

Comment:
I previously added an "email validator" package to the project, so we might as well use it here.